clustered/client/build.gradle

```diff
@@ -22,8 +22,8 @@ dependencies {
   compileOnly project(':api')
   compileOnly project(':xml')
   compile project(':clustered:common')
+  compile "org.slf4j:slf4j-api:$parent.slf4jVersion"
   provided "org.terracotta:entity-client-api:$parent.entityApiVersion"
-  provided "org.terracotta.management:monitoring-service-api:$parent.managementVersion" // provided in management-server jar
```

Comment:
this dependency is the api of a voltron service. should not be on client classpath